gnome-extra/loupe/files/meson-fixes.patch

```diff
@@ -0,0 +1,80 @@
+--- a/src/meson.build
++++ b/src/meson.build
+@@ -15,77 +15,3 @@
+   check: true
+ )
+ 
+-cargo_options = [ '--manifest-path', source_root / 'Cargo.toml' ]
```

Comment:
I'm pretty sure you don't need that, or `CRATES` in ebuild. Look at how I've done glycin-loaders.

Reply:
Done in https://github.com/gentoo/gentoo/commit/7515399cce24e2684aca7a11f4dacccfa1f6777f
Thanks!